docs/citing.rst

```diff
@@ -6,35 +6,159 @@
 Citing FMRIPREP
 ===============
 
-``fmriprep`` is a Nipype_-based pipeline that combines software from several
-neuroimaging packages with custom-written code to flexibly apply
-state-of-the-art preprocessing algorithms to a wide variety of structural and
-functional neuroimages and generate readable reports.
-For a full list of external software and relevant citations (where available)
-of tools and algorithms, please see the `REFERENCES.md
-<https://github.com/poldracklab/fmriprep/blob/master/REFERENCES.md>`_ table in
-the source distribution.
-Please cite the relevant packages to your use case.
-``fmriprep`` may be cited by its Zenodo reference::
-
-    @misc{esteban_oscar_2017_996169,
-      author       = {Esteban, Oscar and
-                      Blair, Ross and
-                      Markiewicz, Christopher J. and
-                      Berleant, Shoshana L. and
-                      Moodie, Craig and
-                      Ma, Feilong and
-                      Isik, Ayse Ilkay and
-                      Erramuzpe, Asier and
-                      Goncalves, Mathias and
-                      Poldrack, Russell A. and
-                      Gorgolewski, Krzysztof J.},
-      title        = {poldracklab/fmriprep: 1.0.0-rc5},
-      month        = sep,
-      year         = 2017,
-      doi          = {10.5281/zenodo.996169},
-      url          = {https://doi.org/10.5281/zenodo.996169}
-    }
+Select which options you have run FMRIPREP with to generate custom language
+we recommend to include in your paper.
+
+
+.. raw:: html
+
+   <script language="javascript">
+   var version = 'latest';
+   function fillCitation(){
+      $('#fmriprep_version').text(version);
+      $('#workflow_url').text('https://fmriprep.readthedocs.io/en/' + version + '/workflows.html');
+      $('#workflow_url').attr('href', 'https://fmriprep.readthedocs.io/en/' + version + '/workflows.html');
+
+      function cb(err, zenodoID) {
+         getCitation(zenodoID, 'vancouver-brackets-no-et-al', function(err, citation) {
+            $('#fmriprep_citation').text(citation);
+         });
+         getDOI(zenodoID, function(err, DOI) {
+            $('#fmriprep_doi_url').text('https://doi.org/' + DOI);
```

Comment:
I usually prefer to hide the https://doi.org/ on DOIs and just show doi:[some-doi-without-http-doi-org](https://doi.org). But it is only a preference.